setup.py

```diff
@@ -30,6 +30,12 @@ def write_version_py(filename=None):
     author='SimPhoNy FP7 European Project',
     description='The native implementation of the SimPhoNy cuds objects',
     long_description=README_TEXT,
+    install_requires=[
+        "cython",
```

Comment:
cood point. cython is actually only necessary for th depedencies not for building simphony-common it-self.